README.md

```diff
@@ -70,15 +84,21 @@ Stop Zendro service.
    
-5. stop all running service by executing **`zendro stop`**. Besides, if you would like to stop SPA with production mode, please add `-p` option.
+5. stop all running service by executing **`zendro stop`**. Besides, if you would like to stop GQS and SPA with production mode, please add `-p` option.
+
+6. If you don't have local database, you can play with Zendro by dockerizing example Zendro App. The command would be **`zendro dockerize -u`**. Moreover, if you would like to use GQS and SPA with production mode, please execute **`zendro dockerize -u -p`**. Besides, the default username is `admin@zen.dro` and the corresponding password is `admin`. 
 
-6. If you don't have local database, you can play with Zendro by dockerizing example Zendro App. The command would be **`zendro dockerize -u`**. Moreover, if you would like to use SPA with production mode, please execute **`zendro dockerize -u -p`**. Besides, the default username is `admin@zen.dro` and the corresponding password is `admin`. 
+7. When you want to stop docker service, press CTRL+C once, then execute **`zendro dockerize -d`**. In addition, if your GQS and SPA is in production mode, please execute **`zendro dockerize -d -p`**. 
 
-7. When you want to stop docker service, press CTRL+C once, then execute **`zendro dockerize -d`**. In addition, if your SPA is in production mode, please execute **`zendro dockerize -d -p`**. 
+### Example for Migrations
+If a user has new data model definitions, it is convinient to use Zendro CLI for dealing with migrations. And the following procedure shows how to generate, perform or drop migrations:
+1. in `graphql-server` folder, execute `zendro migration:generate -f <data_model_definitions>`, then migrations would be generated automatically in `/graphql-server/migrations` folder. By default, every migration file would have two functions, namely `up` and `down` function. In specifically, in `up` function a table or indices would be created and in `down` function the existing table or indices would be deleted. Besides, user can customize their migration functions.
+2. in `graphql-server` folder, it is possible to perform new generated migrations, which are generated after the last executed migration, by executing `zendro migration:up`. After that the last executed migration and migration log would be updated.
+3. in `graphql-server` folder, the last executed migration could be dropped by executing `zendro migration:down`. Then the last executed migration would be updated as the last successfully executed migration before the dropped migration. And the dropped operation would be recorded in the migration log. If there are some remaining records and associations in the table, by default an error would be throwed. If a user still would like to delete all of them, then the user needs set environment variable `DOWN_MIGRATION` to `true` in `/graphql-server/.env` file and re-execute this down-migration. 
```

Comment:
3. in `graphql-server` folder, the last executed migration can be dropped by executing `zendro migration:down`. This will update the latest successful migration and add the dropped operation to the migration log. If there are some remaining records and associations in the table, by default an error is thrown. To forcefully drop the table, in spite of remaining records, set the environment variable `DOWN_MIGRATION` to `true` in `/graphql-server/.env` file and re-execute this down-migration.